
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  </head>
<body ng-app="app">
<div class="container">
        <h2>Metascape Gene List Analysis Report</h2><p/>  <a href="http://metascape.org">metascape.org</a><sup>1</sup><p/>

              <h3>Bar Graph Summary</h3>
              <div class="panel panel-info">
                <div class="panel-heading">Figure 1. Bar graph of enriched terms across input gene lists, colored by p-values.</div>
                <div class="panel-body">

                <table>
                    <tr>
                        <td>
                            <img src="./Enrichment_heatmap/HeatmapSelectedGO.png" style="width:1000px;">
                        </td>
                    </tr>
                    <tr>
                        <td align='center'>
                            
                <a href='./Enrichment_heatmap/HeatmapSelectedGO.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a>
                
                        </td>
                    </tr>
                    
                <tr><td>Metascape only visualizes the top 20 clusters. Up to 100 enriched clusters can be viewed here.<br>
                <a href="./Enrichment_heatmap/HeatmapSelectedGOTop100.png"><img src="./Enrichment_heatmap/HeatmapSelectedGOTop100.png" style="height:auto; width:auto; max-height:100px;"></a>
                <a href='./Enrichment_heatmap/HeatmapSelectedGOTop100.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a></td></tr>
                
                <tr><td>The top-level Gene Ontology biological processes can be viewed here.<br>
                <a href="./Enrichment_heatmap/HeatmapSelectedGOParent.png"><img src="./Enrichment_heatmap/HeatmapSelectedGOParent.png" style="height:auto; width:auto; max-height:100px;"></a>
                <a href='./Enrichment_heatmap/HeatmapSelectedGOParent.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a></td></tr>
                
                </table>
                </div>
              </div><p/>
      <h3>Gene Lists</h3>
      User-provided gene identifiers are first converted into their corresponding H. sapiens Entrez gene IDs using the latest version of the database (last updated on 2022-10-01).  If multiple identifiers correspond to the same Entrez gene ID, they will be considered as a single Entrez gene ID in downstream analyses.   The gene lists are summarized in Table 1.<p/>
          <div class="panel panel-info">
            <div class="panel-heading">Table 1. Statistics of input gene lists.</div>
            <div class="panel-body"><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">Name</TH>
      <TH class="info">Total</TH>
      <TH class="info">Unique</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD>MyList</TD>
      <TD>376</TD>
      <TD>374</TD>
    </TR>
  </TBODY>
</TABLE></div>
          </div>




              <h3>Pathway and Process Enrichment Analysis</h3>
              For each given gene list, pathway and process enrichment analysis has been carried out with the following ontology sources: GO Biological Processes, GO Molecular Functions, GO Cellular Components, KEGG Pathway, Immunologic Signatures, Oncogenic Signatures, Reactome Gene Sets, Canonical Pathways, Chemical And Genetic Perturbations, CORUM, WikiPathways and PANTHER Pathway.  All genes in the genome have been used as the enrichment background. Terms with a p-value &lt; 0.01, a minimum count of 3, and an enrichment factor &gt; 1.5 (the enrichment factor is the ratio between the observed counts and the counts expected by chance) are collected and grouped into clusters based on their membership similarities.  More specifically, p-values are calculated based on the cumulative hypergeometric distribution<sup>2</sup>, and q-values are calculated using the Benjamini-Hochberg procedure to account for multiple testings<sup>3</sup>.  Kappa scores<sup>4</sup> are used as the similarity metric when performing hierarchical clustering on the enriched terms, and sub-trees with a similarity of > 0.3 are considered a cluster.  The most statistically significant term within a cluster is chosen to represent the cluster.<p/>
          <div class="panel panel-info">
          <div class="panel-heading">Table 2. Top 20 clusters with their representative enriched terms (one per cluster).  "Count" is the number of genes in the user-provided lists with membership in the given ontology term. "%" is the percentage of all of the user-provided genes that are found in the given ontology term (only input genes with at least one ontology term annotation are included in the calculation).  "Log10(P)" is the p-value in log base 10.  "Log10(q)" is the multi-test adjusted p-value in log base 10.</div>
                <div class="panel-body"><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">GO</TH>
      <TH class="info">Category</TH>
      <TH class="info">Description</TH>
      <TH class="info">Count</TH>
      <TH class="info">%</TH>
      <TH class="info">Log10(P)</TH>
      <TH class="info">Log10(q)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD>WP2261</TD>
      <TD>WikiPathways</TD>
      <TD>Glioblastoma signaling pathways</TD>
      <TD>33</TD>
      <TD>8.82</TD>
      <TD>-40.61</TD>
      <TD>-36.04</TD>
    </TR>
    <TR>
      <TD>GO:0008283</TD>
      <TD>GO Biological Processes</TD>
      <TD>cell population proliferation</TD>
      <TD>63</TD>
      <TD>16.84</TD>
      <TD>-34.73</TD>
      <TD>-30.94</TD>
    </TR>
    <TR>
      <TD>M15356</TD>
      <TD>Chemical And Genetic Perturbations</TD>
      <TD>PUJANA BREAST CANCER LIT INT NETWORK</TD>
      <TD>29</TD>
      <TD>7.75</TD>
      <TD>-30.88</TD>
      <TD>-27.38</TD>
    </TR>
    <TR>
      <TD>GO:0007507</TD>
      <TD>GO Biological Processes</TD>
      <TD>heart development</TD>
      <TD>53</TD>
      <TD>14.17</TD>
      <TD>-30.28</TD>
      <TD>-26.88</TD>
    </TR>
    <TR>
      <TD>WP5087</TD>
      <TD>WikiPathways</TD>
      <TD>Malignant pleural mesothelioma</TD>
      <TD>46</TD>
      <TD>12.30</TD>
      <TD>-27.99</TD>
      <TD>-24.74</TD>
    </TR>
    <TR>
      <TD>GO:0007423</TD>
      <TD>GO Biological Processes</TD>
      <TD>sensory organ development</TD>
      <TD>50</TD>
      <TD>13.37</TD>
      <TD>-27.12</TD>
      <TD>-23.96</TD>
    </TR>
    <TR>
      <TD>GO:0009725</TD>
      <TD>GO Biological Processes</TD>
      <TD>response to hormone</TD>
      <TD>57</TD>
      <TD>15.24</TD>
      <TD>-27.04</TD>
      <TD>-23.93</TD>
    </TR>
    <TR>
      <TD>GO:0002520</TD>
      <TD>GO Biological Processes</TD>
      <TD>immune system development</TD>
      <TD>56</TD>
      <TD>14.97</TD>
      <TD>-26.69</TD>
      <TD>-23.59</TD>
    </TR>
    <TR>
      <TD>GO:0008134</TD>
      <TD>GO Molecular Functions</TD>
      <TD>transcription factor binding</TD>
      <TD>48</TD>
      <TD>12.83</TD>
      <TD>-24.13</TD>
      <TD>-21.23</TD>
    </TR>
    <TR>
      <TD>M9150</TD>
      <TD>Chemical And Genetic Perturbations</TD>
      <TD>GRESHOCK CANCER COPY NUMBER UP</TD>
      <TD>37</TD>
      <TD>9.89</TD>
      <TD>-23.93</TD>
      <TD>-21.06</TD>
    </TR>
    <TR>
      <TD>WP138</TD>
      <TD>WikiPathways</TD>
      <TD>Androgen receptor signaling pathway</TD>
      <TD>23</TD>
      <TD>6.15</TD>
      <TD>-23.16</TD>
      <TD>-20.40</TD>
    </TR>
    <TR>
      <TD>GO:0009314</TD>
      <TD>GO Biological Processes</TD>
      <TD>response to radiation</TD>
      <TD>41</TD>
      <TD>10.96</TD>
      <TD>-22.79</TD>
      <TD>-20.06</TD>
    </TR>
    <TR>
      <TD>GO:0010564</TD>
      <TD>GO Biological Processes</TD>
      <TD>regulation of cell cycle process</TD>
      <TD>51</TD>
      <TD>13.64</TD>
      <TD>-22.73</TD>
      <TD>-20.00</TD>
    </TR>
    <TR>
      <TD>WP2828</TD>
      <TD>WikiPathways</TD>
      <TD>Bladder cancer</TD>
      <TD>17</TD>
      <TD>4.55</TD>
      <TD>-21.74</TD>
      <TD>-19.11</TD>
    </TR>
    <TR>
      <TD>GO:0043473</TD>
      <TD>GO Biological Processes</TD>
      <TD>pigmentation</TD>
      <TD>23</TD>
      <TD>6.15</TD>
      <TD>-21.40</TD>
      <TD>-18.81</TD>
    </TR>
    <TR>
      <TD>GO:0040007</TD>
      <TD>GO Biological Processes</TD>
      <TD>growth</TD>
      <TD>38</TD>
      <TD>10.16</TD>
      <TD>-21.38</TD>
      <TD>-18.79</TD>
    </TR>
    <TR>
      <TD>GO:0001655</TD>
      <TD>GO Biological Processes</TD>
      <TD>urogenital system development</TD>
      <TD>35</TD>
      <TD>9.36</TD>
      <TD>-20.89</TD>
      <TD>-18.32</TD>
    </TR>
    <TR>
      <TD>GO:0001501</TD>
      <TD>GO Biological Processes</TD>
      <TD>skeletal system development</TD>
      <TD>41</TD>
      <TD>10.96</TD>
      <TD>-20.57</TD>
      <TD>-18.04</TD>
    </TR>
    <TR>
      <TD>GO:0051052</TD>
      <TD>GO Biological Processes</TD>
      <TD>regulation of DNA metabolic process</TD>
      <TD>42</TD>
      <TD>11.23</TD>
      <TD>-20.56</TD>
      <TD>-18.03</TD>
    </TR>
    <TR>
      <TD>GO:0007420</TD>
      <TD>GO Biological Processes</TD>
      <TD>brain development</TD>
      <TD>49</TD>
      <TD>13.10</TD>
      <TD>-20.42</TD>
      <TD>-17.89</TD>
    </TR>
  </TBODY>
</TABLE></div>
              </div><p/>
                  To further capture the relationships between the terms, a subset of enriched terms have been selected and rendered as a network plot, where terms with a similarity &gt; 0.3 are connected by edges.  We select the terms with the best p-values from each of the 20 clusters, with the constraint that there are no more than 15 terms per cluster and no more than 250 terms in total.  The network is visualized using <a href="http://www.cytoscape.org">Cytoscape</a><sup>5</sup>, where each node represents an enriched term and is colored first by its cluster ID (Figure 2.a) and then by its p-value (Figure 2.b).  These networks can be interactively viewed in Cytoscape through the .cys files (contained in the Zip package, which also contains a publication-quality version as a PDF) or within a browser by clicking on the web icon.  For clarity, term labels are only shown for one term per cluster, so it is recommended to use Cytoscape or a browser to visualize the network in order to inspect all node labels.  We can also export the network into a PDF file within Cytoscape, and then edit the labels using Adobe Illustrator for publication purposes.  To switch off all labels, delete the "Label" mapping under the "Style" tab within Cytoscape, and then export the network view.<p/>
                  <div class="panel panel-info">
                    <div class="panel-heading">Figure 2. Network of enriched terms: (a) colored by cluster ID, where nodes that share the same cluster ID are typically close to each other; (b) colored by p-value, where terms containing more genes tend to have a more significant p-value.</div>
                    <div class="panel-body">
                    <table>
                        <tr>
                            <td>
                                <img src="./Enrichment_GO/ColorByCluster.png" style="width:500px;">
                            </td>
                            <td>
                                <img src="./Enrichment_GO/ColorByPValue.png" style="width:500px;">
                            </td>
                        </tr>
                        <tr>
                            <td align='center'>
                                <a href='./Enrichment_GO/ColorByCluster.pdf' title='download PDF file'>
                        <img class='link' src='icon/PDF48.png' >
                        </a>
                     
                        <a href='./Enrichment_GO/GONetwork.cys' title='download CYS file'>
                        <img class='link' src='icon/CYS48.png' >
                        </a>
                    
                        
                    
                        
                        <a href='Enrichment_GO/GONetwork.html?Network=GONetwork&Style=ColorByCluster' title='interactive cytoscape' target='_blank'>
                        <img class='link' src='icon/WEB_CYS48.png' >
                        </a>
                        
                    
                            </td>
                            <td align='center'>
                                <a href='./Enrichment_GO/ColorByPValue.pdf' title='download PDF file'>
                        <img class='link' src='icon/PDF48.png' >
                        </a>
                     
                        <a href='./Enrichment_GO/GONetwork.cys' title='download CYS file'>
                        <img class='link' src='icon/CYS48.png' >
                        </a>
                    
                        
                    
                        
                        <a href='Enrichment_GO/GONetwork.html?Network=GONetwork&Style=ColorByPValue' title='interactive cytoscape' target='_blank'>
                        <img class='link' src='icon/WEB_CYS48.png' >
                        </a>
                        
                    
                            </td>
                        </tr>
                    </table>
                    </div>

                  </div><p/>
<h3>Protein-protein Interaction Enrichment Analysis</h3>
For each given gene list, protein-protein interaction enrichment analysis has been carried out with the following databases: STRING<sup>6</sup>, BioGrid<sup>7</sup>, OmniPath<sup>8</sup>, InWeb_IM<sup>9</sup>.Only physical interactions in STRING (physical score &gt 0.132) and BioGrid are used (<a href="http://metascape.org/blog/?p=219">details</a>).  The resultant network contains the subset of proteins that form physical interactions with at least one other member in the list.  If the network contains between 3 and 500 proteins, the Molecular Complex Detection (MCODE) algorithm<sup>10</sup> has been applied to identify densely connected network components.  The MCODE networks identified for individual gene lists have been gathered and are shown in Figure 3.<p/>

                    Pathway and process enrichment analysis has been applied to each MCODE component independently, and the three best-scoring terms by p-value have been retained as the functional description of the corresponding components, shown in the tables underneath corresponding network plots within Figure 3.<p/>

      <div class="panel panel-info">
        <div class="panel-heading">Figure 3. Protein-protein interaction network and MCODE components identified in the gene lists.</div>
        <div class="panel-body">
        
                <table border-collapse='collapse'>
                <tr><td><img src="./Enrichment_PPI/MyList_PPIColorByCluster.png" style="width:500px;"></td><td width="10px"></td><td><img src="./Enrichment_PPI/MyList_MCODE_ALL_PPIColorByCluster.png" style="width:500px;"></td></tr>
                <tr><td align="center"><a href='./Enrichment_PPI/MyList_PPIColorByCluster.pdf' title='download PDF file'>
                        <img class='link' src='icon/PDF48.png' >
                        </a>
                     
                        <a href='./Enrichment_PPI/MCODE_PPI.cys' title='download CYS file'>
                        <img class='link' src='icon/CYS48.png' >
                        </a>
                    
                        
                    
                        
                        <a href='Enrichment_PPI/PPINetwork.html?Network=MyList_PPIColorByCluster&Style=PPIColorByClusterNoLabel&isPPI=True' title='interactive cytoscape' target='_blank'>
                        <img class='link' src='icon/WEB_CYS48.png' >
                        </a>
                        
                    </td><td width="10px"></td><td align="center"><a href='./Enrichment_PPI/MyList_MCODE_ALL_PPIColorByCluster.pdf' title='download PDF file'>
                        <img class='link' src='icon/PDF48.png' >
                        </a>
                     
                        <a href='./Enrichment_PPI/MCODE_PPI.cys' title='download CYS file'>
                        <img class='link' src='icon/CYS48.png' >
                        </a>
                    
                        
                    
                        
                        <a href='Enrichment_PPI/PPINetwork.html?Network=MyList_MCODE_ALL_PPIColorByCluster&Style=PPIColorByCluster&isPPI=True' title='interactive cytoscape' target='_blank'>
                        <img class='link' src='icon/WEB_CYS48.png' >
                        </a>
                        
                    </td></tr>
                <tr><td align="center" valign="top"><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">GO</TH>
      <TH class="info">Description</TH>
      <TH class="info">Log10(P)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD>WP2261</TD>
      <TD>Glioblastoma signaling pathways</TD>
      <TD>-41.5</TD>
    </TR>
    <TR>
      <TD>hsa05200</TD>
      <TD>Pathways in cancer</TD>
      <TD>-40.2</TD>
    </TR>
    <TR>
      <TD>hsa05215</TD>
      <TD>Prostate cancer</TD>
      <TD>-37.2</TD>
    </TR>
  </TBODY>
</TABLE></td><td width="10px"></td><td align="center" valign="top"><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">Color</TH>
      <TH class="info">MCODE</TH>
      <TH class="info">GO</TH>
      <TH class="info">Description</TH>
      <TH class="info">Log10(P)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD><div style="background-color:#A6CEE3 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_1</TD>
      <TD>R-HSA-9603798</TD>
      <TD>Class I peroxisomal membrane protein import</TD>
      <TD>-18.2</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#A6CEE3 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_1</TD>
      <TD>R-HSA-5620922</TD>
      <TD>BBSome-mediated cargo-targeting to cilium</TD>
      <TD>-17.6</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#A6CEE3 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_1</TD>
      <TD>GO:0034464</TD>
      <TD>BBSome</TD>
      <TD>-15.9</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#1F78B4 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_2</TD>
      <TD>R-HSA-9675136</TD>
      <TD>Diseases of DNA Double-Strand Break Repair</TD>
      <TD>-35.5</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#1F78B4 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_2</TD>
      <TD>R-HSA-9701190</TD>
      <TD>Defective homologous recombination repair (HRR) due to BRCA2 loss of function</TD>
      <TD>-35.5</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#1F78B4 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_2</TD>
      <TD>R-HSA-5693579</TD>
      <TD>Homologous DNA Pairing and Strand Exchange</TD>
      <TD>-35.1</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#B2DF8A !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_3</TD>
      <TD>hsa05215</TD>
      <TD>Prostate cancer</TD>
      <TD>-27.7</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#B2DF8A !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_3</TD>
      <TD>WP306</TD>
      <TD>Focal adhesion</TD>
      <TD>-25.4</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#B2DF8A !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_3</TD>
      <TD>hsa04510</TD>
      <TD>Focal adhesion</TD>
      <TD>-25.3</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#33A02C !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_4</TD>
      <TD>hsa01522</TD>
      <TD>Endocrine resistance</TD>
      <TD>-18.6</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#33A02C !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_4</TD>
      <TD>M115</TD>
      <TD>PID REG GR PATHWAY</TD>
      <TD>-17.0</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#33A02C !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_4</TD>
      <TD>hsa05166</TD>
      <TD>Human T-cell leukemia virus 1 infection</TD>
      <TD>-17.0</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FB9A99 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_5</TD>
      <TD>hsa05200</TD>
      <TD>Pathways in cancer</TD>
      <TD>-15.1</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FB9A99 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_5</TD>
      <TD>WP2446</TD>
      <TD>Retinoblastoma gene in cancer</TD>
      <TD>-12.0</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FB9A99 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_5</TD>
      <TD>WP5087</TD>
      <TD>Malignant pleural mesothelioma</TD>
      <TD>-11.6</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#E31A1C !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_6</TD>
      <TD>GO:0031082</TD>
      <TD>BLOC complex</TD>
      <TD>-20.5</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#E31A1C !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_6</TD>
      <TD>GO:0060155</TD>
      <TD>platelet dense granule organization</TD>
      <TD>-18.9</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#E31A1C !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_6</TD>
      <TD>GO:0032438</TD>
      <TD>melanosome organization</TD>
      <TD>-17.6</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FDBF6F !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_7</TD>
      <TD>GO:0034706</TD>
      <TD>sodium channel complex</TD>
      <TD>-12.4</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FDBF6F !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_7</TD>
      <TD>GO:0005272</TD>
      <TD>sodium channel activity</TD>
      <TD>-11.3</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FDBF6F !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_7</TD>
      <TD>GO:0035725</TD>
      <TD>sodium ion transmembrane transport</TD>
      <TD>-9.5</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FF7F00 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_8</TD>
      <TD>GO:1990779</TD>
      <TD>glycoprotein Ib-IX-V complex</TD>
      <TD>-11.7</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FF7F00 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_8</TD>
      <TD>R-HSA-9673221</TD>
      <TD>Defective F9 activation</TD>
      <TD>-11.4</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FF7F00 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_8</TD>
      <TD>GO:0007597</TD>
      <TD>blood coagulation, intrinsic pathway</TD>
      <TD>-10.9</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#CAB2D6 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_9</TD>
      <TD>R-HSA-9033241</TD>
      <TD>Peroxisomal protein import</TD>
      <TD>-8.1</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#CAB2D6 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_9</TD>
      <TD>hsa04146</TD>
      <TD>Peroxisome</TD>
      <TD>-7.7</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#CAB2D6 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_9</TD>
      <TD>GO:0005777</TD>
      <TD>peroxisome</TD>
      <TD>-7.0</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#6A3D9A !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_10</TD>
      <TD>GO:0036038</TD>
      <TD>MKS complex</TD>
      <TD>-10.2</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#6A3D9A !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_10</TD>
      <TD>GO:0035869</TD>
      <TD>ciliary transition zone</TD>
      <TD>-7.9</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#6A3D9A !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_10</TD>
      <TD>WP4656</TD>
      <TD>Joubert syndrome</TD>
      <TD>-7.8</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FFFF99 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_11</TD>
      <TD>GO:0030321</TD>
      <TD>transepithelial chloride transport</TD>
      <TD>-10.7</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FFFF99 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_11</TD>
      <TD>GO:0070633</TD>
      <TD>transepithelial transport</TD>
      <TD>-9.1</TD>
    </TR>
    <TR>
      <TD><div style="background-color:#FFFF99 !important;width:40px;height:20px;"></div></TD>
      <TD>MCODE_11</TD>
      <TD>GO:0034707</TD>
      <TD>chloride channel complex</TD>
      <TD>-8.3</TD>
    </TR>
  </TBODY>
</TABLE></td></tr>
                </table>
        </div>
      </div><p/>

<h3>Quality Control and Association Analysis</h3>

<p>Gene list enrichments are identified in the following ontology categories: Cell_Type_Signatures, DisGeNET, PaGenBase, TRRUST, Transcription_Factor_Targets. All genes in the genome have been used as the enrichment background. Terms with a p-value &lt; 0.01, a minimum count of 3, and an enrichment factor &gt; 1.5 (the enrichment factor is the ratio between the observed counts and the counts expected by chance) are collected and grouped into clusters based on their membership similarities.  The top few enriched clusters (one term per cluster) are shown in the Figure 4-8.  The algorithm used here is the same as that is used for pathway and process enrichment analysis.<p/>

              <div class="panel panel-info">
                <div class="panel-heading">Figure 4. Summary of enrichment analysis in Cell Type Signatures<sup>11</sup>.</div>
                <div class="panel-body">

                <table>
                    <tr>
                        <td>
                            <img src="./Enrichment_QC/HeatmapSelectedGO_Cell_Type_Signatures.png" style="width:1000px;">
                        </td>
                    </tr>
                    <tr>
                        <td align='center'>
                            
                <a href='./Enrichment_QC/HeatmapSelectedGO_Cell_Type_Signatures.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a>
                
                        </td>
                    </tr>
                    <tr>
                        <td>
                            <br><br><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">GO</TH>
      <TH class="info">Description</TH>
      <TH class="info">Count</TH>
      <TH class="info">%</TH>
      <TH class="info">Log10(P)</TH>
      <TH class="info">Log10(q)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39060" target="_TERM">M39060</a></TD>
      <TD>MANNO MIDBRAIN NEUROTYPES HPROGFPL</TD>
      <TD>18</TD>
      <TD>4.80</TD>
      <TD>-6.80</TD>
      <TD>-5.70</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39175" target="_TERM">M39175</a></TD>
      <TD>MURARO PANCREAS MESENCHYMAL STROMAL CELL</TD>
      <TD>26</TD>
      <TD>7.00</TD>
      <TD>-6.30</TD>
      <TD>-5.20</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39264" target="_TERM">M39264</a></TD>
      <TD>HU FETAL RETINA FIBROBLAST</TD>
      <TD>18</TD>
      <TD>4.80</TD>
      <TD>-5.70</TD>
      <TD>-4.60</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39061" target="_TERM">M39061</a></TD>
      <TD>MANNO MIDBRAIN NEUROTYPES HPROGFPM</TD>
      <TD>17</TD>
      <TD>4.50</TD>
      <TD>-5.60</TD>
      <TD>-4.50</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39108" target="_TERM">M39108</a></TD>
      <TD>AIZARANI LIVER C4 EPCAM POS BILE DUCT CELLS 1</TD>
      <TD>12</TD>
      <TD>3.20</TD>
      <TD>-5.30</TD>
      <TD>-4.20</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39059" target="_TERM">M39059</a></TD>
      <TD>MANNO MIDBRAIN NEUROTYPES HPROGBP</TD>
      <TD>15</TD>
      <TD>4.00</TD>
      <TD>-5.20</TD>
      <TD>-4.20</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39054" target="_TERM">M39054</a></TD>
      <TD>MANNO MIDBRAIN NEUROTYPES HRGL2B</TD>
      <TD>18</TD>
      <TD>4.80</TD>
      <TD>-4.90</TD>
      <TD>-3.90</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39135" target="_TERM">M39135</a></TD>
      <TD>AIZARANI LIVER C39 EPCAM POS BILE DUCT CELLS 4</TD>
      <TD>11</TD>
      <TD>2.90</TD>
      <TD>-4.70</TD>
      <TD>-3.70</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39111" target="_TERM">M39111</a></TD>
      <TD>AIZARANI LIVER C7 EPCAM POS BILE DUCT CELLS 2</TD>
      <TD>12</TD>
      <TD>3.20</TD>
      <TD>-4.70</TD>
      <TD>-3.70</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M41710" target="_TERM">M41710</a></TD>
      <TD>FAN OVARY CL8 MATURE CUMULUS GRANULOSA CELL 2</TD>
      <TD>22</TD>
      <TD>5.90</TD>
      <TD>-4.60</TD>
      <TD>-3.70</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39058" target="_TERM">M39058</a></TD>
      <TD>MANNO MIDBRAIN NEUROTYPES HPROGM</TD>
      <TD>14</TD>
      <TD>3.70</TD>
      <TD>-4.50</TD>
      <TD>-3.50</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39257" target="_TERM">M39257</a></TD>
      <TD>MENON FETAL KIDNEY 7 LOOPOF HENLE CELLS DISTAL</TD>
      <TD>13</TD>
      <TD>3.50</TD>
      <TD>-4.30</TD>
      <TD>-3.40</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39057" target="_TERM">M39057</a></TD>
      <TD>MANNO MIDBRAIN NEUROTYPES HRGL1</TD>
      <TD>15</TD>
      <TD>4.00</TD>
      <TD>-4.20</TD>
      <TD>-3.30</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M40175" target="_TERM">M40175</a></TD>
      <TD>DESCARTES FETAL EYE CORNEAL AND CONJUNCTIVAL EPITHELIAL CELLS</TD>
      <TD>12</TD>
      <TD>3.20</TD>
      <TD>-4.20</TD>
      <TD>-3.30</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39224" target="_TERM">M39224</a></TD>
      <TD>LAKE ADULT KIDNEY C5 PROXIMAL TUBULE EPITHELIAL CELLS STRESS INFLAM</TD>
      <TD>16</TD>
      <TD>4.30</TD>
      <TD>-4.10</TD>
      <TD>-3.20</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M41675" target="_TERM">M41675</a></TD>
      <TD>TRAVAGLINI LUNG ADVENTITIAL FIBROBLAST CELL</TD>
      <TD>13</TD>
      <TD>3.50</TD>
      <TD>-4.00</TD>
      <TD>-3.10</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39209" target="_TERM">M39209</a></TD>
      <TD>HAY BONE MARROW STROMAL</TD>
      <TD>23</TD>
      <TD>6.10</TD>
      <TD>-4.00</TD>
      <TD>-3.10</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39077" target="_TERM">M39077</a></TD>
      <TD>ZHONG PFC MAJOR TYPES MICROGLIA</TD>
      <TD>16</TD>
      <TD>4.30</TD>
      <TD>-3.90</TD>
      <TD>-3.00</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39246" target="_TERM">M39246</a></TD>
      <TD>LAKE ADULT KIDNEY C27 VASCULAR SMOOTH MUSCLE CELLS AND PERICYTES</TD>
      <TD>9</TD>
      <TD>2.40</TD>
      <TD>-3.80</TD>
      <TD>-2.90</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M39129" target="_TERM">M39129</a></TD>
      <TD>AIZARANI LIVER C30 HEPATOCYTES 4</TD>
      <TD>16</TD>
      <TD>4.30</TD>
      <TD>-3.80</TD>
      <TD>-2.90</TD>
    </TR>
  </TBODY>
</TABLE>
                        </td>
                    </tr>
                </table>

                </div>
              </div>

              <div class="panel panel-info">
                <div class="panel-heading">Figure 5. Summary of enrichment analysis in DisGeNET<sup>12</sup>.</div>
                <div class="panel-body">

                <table>
                    <tr>
                        <td>
                            <img src="./Enrichment_QC/HeatmapSelectedGO_DisGeNET.png" style="width:1000px;">
                        </td>
                    </tr>
                    <tr>
                        <td align='center'>
                            
                <a href='./Enrichment_QC/HeatmapSelectedGO_DisGeNET.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a>
                
                        </td>
                    </tr>
                    <tr>
                        <td>
                            <br><br><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">GO</TH>
      <TH class="info">Description</TH>
      <TH class="info">Count</TH>
      <TH class="info">%</TH>
      <TH class="info">Log10(P)</TH>
      <TH class="info">Log10(q)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C1857276/" target="_TERM">C1857276</a></TD>
      <TD>Trichohepatoenteric Syndrome</TD>
      <TD>88</TD>
      <TD>24.00</TD>
      <TD>-81.00</TD>
      <TD>-77.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C1384666/" target="_TERM">C1384666</a></TD>
      <TD>hearing impairment</TD>
      <TD>104</TD>
      <TD>28.00</TD>
      <TD>-79.00</TD>
      <TD>-74.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0038379/" target="_TERM">C0038379</a></TD>
      <TD>Strabismus</TD>
      <TD>100</TD>
      <TD>27.00</TD>
      <TD>-75.00</TD>
      <TD>-71.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0010417/" target="_TERM">C0010417</a></TD>
      <TD>Cryptorchidism</TD>
      <TD>99</TD>
      <TD>26.00</TD>
      <TD>-73.00</TD>
      <TD>-69.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0033377/" target="_TERM">C0033377</a></TD>
      <TD>Ptosis</TD>
      <TD>91</TD>
      <TD>24.00</TD>
      <TD>-71.00</TD>
      <TD>-67.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0005745/" target="_TERM">C0005745</a></TD>
      <TD>Blepharoptosis</TD>
      <TD>90</TD>
      <TD>24.00</TD>
      <TD>-70.00</TD>
      <TD>-67.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0025990/" target="_TERM">C0025990</a></TD>
      <TD>Micrognathism</TD>
      <TD>89</TD>
      <TD>24.00</TD>
      <TD>-70.00</TD>
      <TD>-66.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C1963184/" target="_TERM">C1963184</a></TD>
      <TD>Nystagmus, CTCAE 3.0</TD>
      <TD>95</TD>
      <TD>25.00</TD>
      <TD>-65.00</TD>
      <TD>-62.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C4554036/" target="_TERM">C4554036</a></TD>
      <TD>Nystagmus, CTCAE 5.0</TD>
      <TD>95</TD>
      <TD>25.00</TD>
      <TD>-65.00</TD>
      <TD>-62.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0020534/" target="_TERM">C0020534</a></TD>
      <TD>Orbital separation excessive</TD>
      <TD>82</TD>
      <TD>22.00</TD>
      <TD>-61.00</TD>
      <TD>-57.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0239234/" target="_TERM">C0239234</a></TD>
      <TD>Low set ears</TD>
      <TD>76</TD>
      <TD>20.00</TD>
      <TD>-60.00</TD>
      <TD>-56.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0018784/" target="_TERM">C0018784</a></TD>
      <TD>Sensorineural Hearing Loss (disorder)</TD>
      <TD>90</TD>
      <TD>24.00</TD>
      <TD>-60.00</TD>
      <TD>-56.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0240635/" target="_TERM">C0240635</a></TD>
      <TD>Byzanthine arch palate</TD>
      <TD>75</TD>
      <TD>20.00</TD>
      <TD>-58.00</TD>
      <TD>-55.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0008925/" target="_TERM">C0008925</a></TD>
      <TD>Cleft Palate</TD>
      <TD>79</TD>
      <TD>21.00</TD>
      <TD>-56.00</TD>
      <TD>-52.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0423110/" target="_TERM">C0423110</a></TD>
      <TD>Downward slant of palpebral fissure</TD>
      <TD>66</TD>
      <TD>18.00</TD>
      <TD>-54.00</TD>
      <TD>-51.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0848558/" target="_TERM">C0848558</a></TD>
      <TD>Hypospadias</TD>
      <TD>64</TD>
      <TD>17.00</TD>
      <TD>-53.00</TD>
      <TD>-50.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0018818/" target="_TERM">C0018818</a></TD>
      <TD>Ventricular Septal Defects</TD>
      <TD>65</TD>
      <TD>17.00</TD>
      <TD>-50.00</TD>
      <TD>-47.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C1836542/" target="_TERM">C1836542</a></TD>
      <TD>Depressed nasal bridge</TD>
      <TD>63</TD>
      <TD>17.00</TD>
      <TD>-48.00</TD>
      <TD>-45.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C3887461/" target="_TERM">C3887461</a></TD>
      <TD>Head and Neck Carcinoma</TD>
      <TD>79</TD>
      <TD>21.00</TD>
      <TD>-47.00</TD>
      <TD>-44.00</TD>
    </TR>
    <TR>
      <TD><a href="http://www.disgenet.org/browser/0/1/2/C0020255/" target="_TERM">C0020255</a></TD>
      <TD>Hydrocephalus</TD>
      <TD>65</TD>
      <TD>17.00</TD>
      <TD>-47.00</TD>
      <TD>-44.00</TD>
    </TR>
  </TBODY>
</TABLE>
                        </td>
                    </tr>
                </table>

                </div>
              </div>

              <div class="panel panel-info">
                <div class="panel-heading">Figure 6. Summary of enrichment analysis in PaGenBase<sup>13</sup>.</div>
                <div class="panel-body">

                <table>
                    <tr>
                        <td>
                            <img src="./Enrichment_QC/HeatmapSelectedGO_PaGenBase.png" style="width:1000px;">
                        </td>
                    </tr>
                    <tr>
                        <td align='center'>
                            
                <a href='./Enrichment_QC/HeatmapSelectedGO_PaGenBase.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a>
                
                        </td>
                    </tr>
                    <tr>
                        <td>
                            <br><br><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">GO</TH>
      <TH class="info">Description</TH>
      <TH class="info">Count</TH>
      <TH class="info">%</TH>
      <TH class="info">Log10(P)</TH>
      <TH class="info">Log10(q)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD>PGB:00015</TD>
      <TD>Tissue-specific: Smooth Muscle</TD>
      <TD>9</TD>
      <TD>2.40</TD>
      <TD>-5.20</TD>
      <TD>-4.20</TD>
    </TR>
    <TR>
      <TD>PGB:00036</TD>
      <TD>Cell-specific: MOLT4</TD>
      <TD>10</TD>
      <TD>2.70</TD>
      <TD>-5.10</TD>
      <TD>-4.00</TD>
    </TR>
    <TR>
      <TD>PGB:00082</TD>
      <TD>Cell-specific: Breast cell</TD>
      <TD>6</TD>
      <TD>1.60</TD>
      <TD>-4.20</TD>
      <TD>-3.30</TD>
    </TR>
    <TR>
      <TD>PGB:00120</TD>
      <TD>Cell-specific: Cardiac Myocytes</TD>
      <TD>6</TD>
      <TD>1.60</TD>
      <TD>-4.00</TD>
      <TD>-3.10</TD>
    </TR>
    <TR>
      <TD>PGB:00004</TD>
      <TD>Tissue-specific: kidney</TD>
      <TD>16</TD>
      <TD>4.30</TD>
      <TD>-3.40</TD>
      <TD>-2.50</TD>
    </TR>
    <TR>
      <TD>PGB:00047</TD>
      <TD>Cell-specific: THY+</TD>
      <TD>7</TD>
      <TD>1.90</TD>
      <TD>-3.30</TD>
      <TD>-2.50</TD>
    </TR>
    <TR>
      <TD>PGB:00041</TD>
      <TD>Tissue-specific: Blood</TD>
      <TD>10</TD>
      <TD>2.70</TD>
      <TD>-3.10</TD>
      <TD>-2.20</TD>
    </TR>
    <TR>
      <TD>PGB:00054</TD>
      <TD>Tissue-specific: thyroid</TD>
      <TD>9</TD>
      <TD>2.40</TD>
      <TD>-2.60</TD>
      <TD>-1.80</TD>
    </TR>
    <TR>
      <TD>PGB:00025</TD>
      <TD>Tissue-specific: pancreas</TD>
      <TD>9</TD>
      <TD>2.40</TD>
      <TD>-2.40</TD>
      <TD>-1.70</TD>
    </TR>
    <TR>
      <TD>PGB:00014</TD>
      <TD>Cell-specific: DRG</TD>
      <TD>13</TD>
      <TD>3.50</TD>
      <TD>-2.30</TD>
      <TD>-1.50</TD>
    </TR>
    <TR>
      <TD>PGB:00048</TD>
      <TD>Tissue-specific: bone marrow</TD>
      <TD>8</TD>
      <TD>2.10</TD>
      <TD>-2.10</TD>
      <TD>-1.30</TD>
    </TR>
    <TR>
      <TD>PGB:00063</TD>
      <TD>Cell-specific: OVR278S</TD>
      <TD>4</TD>
      <TD>1.10</TD>
      <TD>-2.00</TD>
      <TD>-1.30</TD>
    </TR>
    <TR>
      <TD>PGB:00067</TD>
      <TD>Cell-specific: Brain cell</TD>
      <TD>4</TD>
      <TD>1.10</TD>
      <TD>-2.00</TD>
      <TD>-1.30</TD>
    </TR>
  </TBODY>
</TABLE>
                        </td>
                    </tr>
                </table>

                </div>
              </div>

              <div class="panel panel-info">
                <div class="panel-heading">Figure 7. Summary of enrichment analysis in TRRUST.</div>
                <div class="panel-body">

                <table>
                    <tr>
                        <td>
                            <img src="./Enrichment_QC/HeatmapSelectedGO_TRRUST.png" style="width:1000px;">
                        </td>
                    </tr>
                    <tr>
                        <td align='center'>
                            
                <a href='./Enrichment_QC/HeatmapSelectedGO_TRRUST.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a>
                
                        </td>
                    </tr>
                    <tr>
                        <td>
                            <br><br><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">GO</TH>
      <TH class="info">Description</TH>
      <TH class="info">Count</TH>
      <TH class="info">%</TH>
      <TH class="info">Log10(P)</TH>
      <TH class="info">Log10(q)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD>TRR01256</TD>
      <TD>Regulated by: SP1</TD>
      <TD>46</TD>
      <TD>12.00</TD>
      <TD>-25.00</TD>
      <TD>-23.00</TD>
    </TR>
    <TR>
      <TD>TRR00275</TD>
      <TD>Regulated by: ESR1</TD>
      <TD>21</TD>
      <TD>5.60</TD>
      <TD>-21.00</TD>
      <TD>-19.00</TD>
    </TR>
    <TR>
      <TD>TRR01419</TD>
      <TD>Regulated by: TP53</TD>
      <TD>24</TD>
      <TD>6.40</TD>
      <TD>-17.00</TD>
      <TD>-16.00</TD>
    </TR>
    <TR>
      <TD>TRR00875</TD>
      <TD>Regulated by: NFKB1</TD>
      <TD>30</TD>
      <TD>8.00</TD>
      <TD>-16.00</TD>
      <TD>-15.00</TD>
    </TR>
    <TR>
      <TD>TRR01158</TD>
      <TD>Regulated by: RELA</TD>
      <TD>29</TD>
      <TD>7.80</TD>
      <TD>-16.00</TD>
      <TD>-14.00</TD>
    </TR>
    <TR>
      <TD>TRR00075</TD>
      <TD>Regulated by: BRCA1</TD>
      <TD>15</TD>
      <TD>4.00</TD>
      <TD>-15.00</TD>
      <TD>-14.00</TD>
    </TR>
    <TR>
      <TD>TRR01277</TD>
      <TD>Regulated by: STAT3</TD>
      <TD>20</TD>
      <TD>5.30</TD>
      <TD>-14.00</TD>
      <TD>-12.00</TD>
    </TR>
    <TR>
      <TD>TRR00230</TD>
      <TD>Regulated by: E2F1</TD>
      <TD>19</TD>
      <TD>5.10</TD>
      <TD>-14.00</TD>
      <TD>-12.00</TD>
    </TR>
    <TR>
      <TD>TRR00011</TD>
      <TD>Regulated by: AR</TD>
      <TD>17</TD>
      <TD>4.50</TD>
      <TD>-14.00</TD>
      <TD>-12.00</TD>
    </TR>
    <TR>
      <TD>TRR00253</TD>
      <TD>Regulated by: EGR1</TD>
      <TD>15</TD>
      <TD>4.00</TD>
      <TD>-12.00</TD>
      <TD>-10.00</TD>
    </TR>
    <TR>
      <TD>TRR01012</TD>
      <TD>Regulated by: PARP1</TD>
      <TD>9</TD>
      <TD>2.40</TD>
      <TD>-11.00</TD>
      <TD>-9.30</TD>
    </TR>
    <TR>
      <TD>TRR00466</TD>
      <TD>Regulated by: HDAC1</TD>
      <TD>13</TD>
      <TD>3.50</TD>
      <TD>-11.00</TD>
      <TD>-9.20</TD>
    </TR>
    <TR>
      <TD>TRR01529</TD>
      <TD>Regulated by: WT1</TD>
      <TD>12</TD>
      <TD>3.20</TD>
      <TD>-10.00</TD>
      <TD>-8.90</TD>
    </TR>
    <TR>
      <TD>TRR00645</TD>
      <TD>Regulated by: JUN</TD>
      <TD>17</TD>
      <TD>4.50</TD>
      <TD>-10.00</TD>
      <TD>-8.90</TD>
    </TR>
    <TR>
      <TD>TRR00780</TD>
      <TD>Regulated by: MYC</TD>
      <TD>14</TD>
      <TD>3.70</TD>
      <TD>-9.90</TD>
      <TD>-8.60</TD>
    </TR>
    <TR>
      <TD>TRR00270</TD>
      <TD>Regulated by: EP300</TD>
      <TD>11</TD>
      <TD>2.90</TD>
      <TD>-9.60</TD>
      <TD>-8.20</TD>
    </TR>
    <TR>
      <TD>TRR01437</TD>
      <TD>Regulated by: TWIST1</TD>
      <TD>9</TD>
      <TD>2.40</TD>
      <TD>-9.20</TD>
      <TD>-7.90</TD>
    </TR>
    <TR>
      <TD>TRR01259</TD>
      <TD>Regulated by: SP3</TD>
      <TD>14</TD>
      <TD>3.70</TD>
      <TD>-9.20</TD>
      <TD>-7.90</TD>
    </TR>
    <TR>
      <TD>TRR00276</TD>
      <TD>Regulated by: ESR2</TD>
      <TD>7</TD>
      <TD>1.90</TD>
      <TD>-8.20</TD>
      <TD>-6.90</TD>
    </TR>
    <TR>
      <TD>TRR01519</TD>
      <TD>Regulated by: VDR</TD>
      <TD>9</TD>
      <TD>2.40</TD>
      <TD>-8.20</TD>
      <TD>-6.90</TD>
    </TR>
  </TBODY>
</TABLE>
                        </td>
                    </tr>
                </table>

                </div>
              </div>

              <div class="panel panel-info">
                <div class="panel-heading">Figure 8. Summary of enrichment analysis in Transcription Factor Targets.</div>
                <div class="panel-body">

                <table>
                    <tr>
                        <td>
                            <img src="./Enrichment_QC/HeatmapSelectedGO_Transcription_Factor_Targets.png" style="width:1000px;">
                        </td>
                    </tr>
                    <tr>
                        <td align='center'>
                            
                <a href='./Enrichment_QC/HeatmapSelectedGO_Transcription_Factor_Targets.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a>
                
                        </td>
                    </tr>
                    <tr>
                        <td>
                            <br><br><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">GO</TH>
      <TH class="info">Description</TH>
      <TH class="info">Count</TH>
      <TH class="info">%</TH>
      <TH class="info">Log10(P)</TH>
      <TH class="info">Log10(q)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M13849" target="_TERM">M13849</a></TD>
      <TD>TGACATY UNKNOWN</TD>
      <TD>30</TD>
      <TD>8.00</TD>
      <TD>-8.70</TD>
      <TD>-7.40</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M17420" target="_TERM">M17420</a></TD>
      <TD>WGGAATGY TEF1 Q6</TD>
      <TD>20</TD>
      <TD>5.30</TD>
      <TD>-7.00</TD>
      <TD>-5.80</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M11968" target="_TERM">M11968</a></TD>
      <TD>NKX61 01</TD>
      <TD>15</TD>
      <TD>4.00</TD>
      <TD>-6.50</TD>
      <TD>-5.40</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M6885" target="_TERM">M6885</a></TD>
      <TD>AREB6 02</TD>
      <TD>15</TD>
      <TD>4.00</TD>
      <TD>-6.00</TD>
      <TD>-4.90</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M11607" target="_TERM">M11607</a></TD>
      <TD>AAAYRNCTG UNKNOWN</TD>
      <TD>18</TD>
      <TD>4.80</TD>
      <TD>-5.90</TD>
      <TD>-4.80</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M16207" target="_TERM">M16207</a></TD>
      <TD>AP2 Q6 01</TD>
      <TD>15</TD>
      <TD>4.00</TD>
      <TD>-5.80</TD>
      <TD>-4.70</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M16280" target="_TERM">M16280</a></TD>
      <TD>PITX2 Q2</TD>
      <TD>14</TD>
      <TD>3.70</TD>
      <TD>-5.20</TD>
      <TD>-4.20</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M15183" target="_TERM">M15183</a></TD>
      <TD>E12 Q6</TD>
      <TD>14</TD>
      <TD>3.70</TD>
      <TD>-5.20</TD>
      <TD>-4.10</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M30042" target="_TERM">M30042</a></TD>
      <TD>KMT2D TARGET GENES</TD>
      <TD>26</TD>
      <TD>7.00</TD>
      <TD>-5.10</TD>
      <TD>-4.10</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M2490" target="_TERM">M2490</a></TD>
      <TD>STAT6 01</TD>
      <TD>14</TD>
      <TD>3.70</TD>
      <TD>-5.10</TD>
      <TD>-4.00</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M30019" target="_TERM">M30019</a></TD>
      <TD>HSD17B8 TARGET GENES</TD>
      <TD>23</TD>
      <TD>6.10</TD>
      <TD>-5.00</TD>
      <TD>-4.00</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M946" target="_TERM">M946</a></TD>
      <TD>GGATTA PITX2 Q2</TD>
      <TD>21</TD>
      <TD>5.60</TD>
      <TD>-4.70</TD>
      <TD>-3.70</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M19368" target="_TERM">M19368</a></TD>
      <TD>TGATTTRY GFI1 01</TD>
      <TD>14</TD>
      <TD>3.70</TD>
      <TD>-4.60</TD>
      <TD>-3.70</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M30363" target="_TERM">M30363</a></TD>
      <TD>ZNF596 TARGET GENES</TD>
      <TD>22</TD>
      <TD>5.90</TD>
      <TD>-4.50</TD>
      <TD>-3.60</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M16694" target="_TERM">M16694</a></TD>
      <TD>AP4 01</TD>
      <TD>13</TD>
      <TD>3.50</TD>
      <TD>-4.50</TD>
      <TD>-3.50</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M6422" target="_TERM">M6422</a></TD>
      <TD>TTAYRTAA E4BP4 01</TD>
      <TD>13</TD>
      <TD>3.50</TD>
      <TD>-4.50</TD>
      <TD>-3.50</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M17644" target="_TERM">M17644</a></TD>
      <TD>OSF2 Q6</TD>
      <TD>13</TD>
      <TD>3.50</TD>
      <TD>-4.40</TD>
      <TD>-3.50</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M16022" target="_TERM">M16022</a></TD>
      <TD>CTAWWWATA RSRFC4 Q2</TD>
      <TD>15</TD>
      <TD>4.00</TD>
      <TD>-4.20</TD>
      <TD>-3.30</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M12580" target="_TERM">M12580</a></TD>
      <TD>GATA3 01</TD>
      <TD>12</TD>
      <TD>3.20</TD>
      <TD>-4.20</TD>
      <TD>-3.30</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M9955" target="_TERM">M9955</a></TD>
      <TD>TGAYRTCA ATF3 Q6</TD>
      <TD>19</TD>
      <TD>5.10</TD>
      <TD>-4.20</TD>
      <TD>-3.20</TD>
    </TR>
  </TBODY>
</TABLE>
                        </td>
                    </tr>
                </table>

                </div>
              </div>
    <h3>Reference</h3></p>
    <ol style="list-style: decimal inside;">
      <li>Zhou et al., Metascape provides a biologist-oriented resource for the analysis of systems-level datasets. Nature Communications (2019) 10(1):1523.</li>
      <li>Zar, J.H. Biostatistical Analysis 1999 4th edn., NJ Prentice Hall, pp. 523</li>
      <li>Hochberg Y., Benjamini Y. More powerful procedures for multiple significance testing. Statistics in Medicine (1990) 9:811-818.</li>
      <li>Cohen, J. A coefficient of agreement for nominal scales. Educ. Psychol. Meas. (1960) 20:27-46.</li>
      <li>Shannon P. et al., Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res (2003) 11:2498-2504.</li>
      <li>Szklarczyk D. et al. STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. (2019) 47:D607-613.</li>
      <li>Stark C. et al. BioGRID: a general repository for interaction datasets. Nucleic Acids Res. (2006) 34:D535-539.</li>
      <li>Turei D. et al. A scored human protein-protein interaction network to catalyze genomic interpretation. Nat. Methods. (2016) 13:966-967.</li>
      <li>Li T. et al. A scored human protein-protein interaction network to catalyze genomic interpretation. Nat. Methods. (2017) 14:61-64.</li>
      <li>Bader, G.D. et al. An automated method for finding molecular complexes in large protein interaction networks. BMC bioinformatics (2003) 4:2.</li>
      <li>Subramanian A, et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A 102, 15545-15550 (2005).</li>
      <li>Pinero J, et al. DisGeNET: a comprehensive platform integrating information on human disease-associated genes and variants. Nucleic acids research 45, D833-D839 (2017).</li>
      <li>Pan JB, et al. PaGenBase: a pattern gene database for the global and dynamic understanding of gene function. PLoS One 8, e80747 (2013).</li>
</ol>
</div>

</body></html>